docs/networking/harvester-network.md

```diff
@@ -58,7 +62,7 @@ You can now create a new VM using the VLAN network configured above:
 
 ## Untagged Network
 
-As is known, the traffic under a VLAN network has a VLAN ID tag and we can use the VLAN network with `PVID` (default: 1) to communicate with any normal untagged traffic. However, some network devices may not be able to receive an explicitly tagged VLAN ID that matches the native/default VLAN on the switch the uplink belongs to. That's the reason why we provide the untagged network.
+As is known, the traffic under a VLAN network has a VLAN ID tag and we can use the VLAN network with `PVID` (default 1) to communicate with any normal untagged traffic. However, some network devices may not expect to receive an explicitly tagged VLAN ID that matches the native VLAN on the switch the uplink belongs to. That's the reason why we provide the untagged network.
 
 ### How to use untagged network
 The way to use an untagged network is similar to the VLAN network. Before adding a new untagged network device for the VM, you have to create a new one if you don't already have one.
```

Comment:
`The way to use an untagged network is similar to the VLAN network` -> `The usage of untagged network is simliar to the VLAN network`.

suggest to remove 
<del> Before adding a new untagged network device for the VM, you have to create a new one if you don't already have one. 
</de> 

and add a link to something link`how to use VLAN network`.